static/templates/stream_settings/stream_types.hbs

```diff
@@ -61,11 +66,9 @@
             <label class="inline-block">
                 {{t 'N' }}:
             </label>
-            <input type="text" autocomplete="off"
-              name="stream-message-retention-days"
-              class="stream-message-retention-days"
-              value="{{ stream_message_retention_days }}"
-              {{#if disable_message_retention_setting}}disabled{{/if}}/>
+            <input type="text" autocomplete="off" name="stream-message-retention-days"
+              class="stream-message-retention-days" value="{{ stream_message_retention_days }}" {{#if
+              disable_message_retention_setting}}disabled{{/if}} />
```

Comment:
Why you have done these changes? Are you using any other linter that we don't use in Zulip?

Reply:
I did run the lint test and the tests got passed. I guess the lint test isn't able to identify the differences. BTW i have made some lint fixes in the updates commit.

Reply:
Got it. I have updated the commits according to what i think would be suitable keeping readability and consistency in my. Please share you feedback.

Reply:
Thanks. But we should have those lint-only changes (only if we consider it as an improvement) in a separate commit explaining that this has been changed for better readability as I pointed out in above comment as those changes are not related to the others UI improvements that this PR is focussing on.

Reply:
Yeah, I agree, please do a `git rebase -i` and clean up which changes are in which commits, moving all the non-functional changes to a separate commit from the functional ones. This PR is quite hard to read as it is written, even using tools like `git show --color-words`